airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4706,6 +4762,31 @@ components:
           $ref: "#/components/schemas/ResourceId"
         workspaceId:
           $ref: "#/components/schemas/WorkspaceId"
+    AttemptId:
+      type: integer
+      format: int32
```

Comment:
I remember commenting about this being an int64: https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L3915

Long term we should merge these types. i.e. `AttemptRead` should use `AttemptId`. I think for now we can standardise on int64.

Reply:
Xiaohan and I just discussed this, there is some confusion here between an `attempt id` and an `attempt number`. We actually want to use the `attempt_number` here, as you can see by the underlying persistence method that our new API calls:
![image](https://user-images.githubusercontent.com/7798112/186785921-8172787b-984c-4d2e-a09e-ae1ff5986bc9.png)

I'll modify the `config.yaml` here to make this clear!